Content.Shared/Speech/Components/VulgarAccentComponent.cs

```diff
@@ -1,10 +1,16 @@
 using Robust.Shared.GameStates;
+using Content.Shared.Dataset;
+using Robust.Shared.Prototypes;
 
 namespace Content.Shared.Speech.Components;
 
 [RegisterComponent, NetworkedComponent]
 public sealed partial class VulgarAccentComponent : Component
 {
+    [DataField]
+    public ProtoId<LocalizedDatasetPrototype> Pack { get; private set; }
+    = "SwearWords";
```

Comment:
i may be wrong with it, but in this case you just can do this
```suggestion
    [DataField]
    public ProtoId<LocalizedDatasetPrototype> Pack = "SwearWords";
```
then i'm leaving the converation because localizing is fine, thanks